### ISSUE TITLE
Cross-posting blog posts

### ISSUE THREAD

**Issue by kellieotto** [rlbarter/Practical-Statistics#4]
There's a bunch of interest from others in what we're doing and people want to collaborate (or use our work for their credit ;) ).  One suggestion was to cross-post on the BIDS blog.  Another suggestion was that some undergrads are trying to start an online publication called Berkeley Data Science Review and they might be interested in publishing some of our stuff.

On the one hand, I like the idea of more exposure.  On the other hand, we will _really_ need to think about our audience.  And part of me doesn't want to touch the undergrad data science curriculum stuff with a 10 foot pole.

Thoughts?

**Comment by rlbarter** [comment]
Well I think the more places we can get our stuff the better, but people need to be aware that their stuff may be posted elsewhere. 

I like the idea of the undergrad one actually, because I'm sure a lot of undergrads would find having accessible summaries on topics that they are just learning very helpful!

**Comment by kellieotto** [comment]
Agreed that it would be helpful to more people.  Maybe I'm just cynical about the amount of extra effort it'll take.  

Didn't want to flood inboxes but someone also brought up the video idea.  We needn't reinvent the wheel if the BIDS ETWG starts doing Khan Academy style videos -- we can see who here is interested in joining in.

**Comment by rlbarter** [comment]
Yes, but if it literally just involves sending them the already-written blog post so that they can cross-post it on their page, I don't see where the extra time and effort would come into place :).

We can bring the video idea up at the meeting, but yes, I don't imagine that we will have the necessary enthusiasm required to do all three of a presentation, blog post and separate video (although if each were done by a different person... maybe).
